README.md

```diff
@@ -1,16 +1,31 @@
-# AI RHDH Installer
+# Red Hat Developer Hub Installer
```

Comment:
I think 'Red Hat Developer Hub Installer' might be too general to RHDH rather than to a varent for AI Software Templates, maybe 'AI Red Hat Developer Hub Installer' or 'Red Hat Developer Hub Installer for AI Software Templates' might sound more specific. WDYT?

Reply:
Yeah I like 'Red Hat Developer Hub Installer for AI Software Templates', identifies this to more what it is.